tests/e2e/e2e_test.go

```diff
@@ -14,7 +14,45 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/util"
 )
 
-func (s *IntegrationTestSuite) TestQueryBalances() {
+func (s *IntegrationTestSuite) Test1IBCTokenTransfer() {
```

Comment:
Let's name `001`. I'm just expecting that we might have more tests relying on order soon